### PR TITLE
HTML

### PR DESCRIPTION
**Pull request by zyelcrier**

### CODE REVIEW

**Comment by ljfranklin** [comment]
@zyelcrier I can't add rayliu0317 as a Reviewer, can you make sure you've invited him to this repo and that he's accepted the invite?

File: portfolio/src/main/webapp/Anime.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      </nav>
+      <div>
+          <h1>List Of My Favourites</h1>
+          <ul>
+              <li>
+                  <h3>Ouran Highschool Host Club</h3>
+                  <p>
+                      Throwback to middleschool weekends with my siblings<br>
+                      This anime was hilarious and great for unwinding and relaxing with the sibs
+                      </p>
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/Anime.html:26]: In the cloud shell editor, right click in the main panel and select Format Document to fix these indents.

File: portfolio/src/main/webapp/Anime.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                  </p>
+                  <img src="Anime/neoyokio.png"/>
+              </li>
+              <li>
+                  <h3>Saga Of Tanya The Evil</h3>
+                  <p>
+                      This was a reccomendation from my brother he said I reminded him of Tanya...
+                      I cant say I agree with his assement.. but I cant say I dont disagree either
+                  </p>
+                  <img src="Anime/sagaof tanya.jpg"/>
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/Anime.html:52]: Avoid spaces in filepaths, it forces you to wrap paths in quotes when you're in the terminal. Use underscores or hyphens instead, just be consistent in using one or the other.

File: portfolio/src/main/webapp/Anime.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              </li>
+              <li>
+                  <h3>Baka and Test</h3>
+                  <p>
+                      I wish school was like this anime. I would have loved Highschool
+                      if this was how it was.
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/Anime.html:58]: Make sure your sentences consistently either end in a period or not

File: portfolio/src/main/webapp/Anime.html (hunk 2 above)
Comment (zyelcrier) [reply portfolio/src/main/webapp/Anime.html:52]: which space?

Comment (ljfranklin) [reply portfolio/src/main/webapp/Anime.html:52]: Space between `sagaof` and `tanya.jpg`

Comment (zyelcrier) [reply portfolio/src/main/webapp/Anime.html]: oh I see i did that because there was a space in the image file so i renamed the file and fixed it

**Comment by zyelcrier** [comment]
> Can you try running the `prettier` command on all these files if you haven't already? You currently have a mix of tabs and spaces in each file so the indents look way off. Try viewing on Github: https://github.com/zyelcrier/step/pull/3/files.
@ljfranklin 
Can we go over this in our meeting because I havent been able to figure it out and I feel it would be most useful to run through it with you

**Comment by ljfranklin** [comment]
@zyelcrier in future PRs try to make the title more descriptive. From my previous comment, a good commit message / PR title might be:

```
Replaced Work page with Anime page

- Additional notes go here
- More notes if you want
```

**Pull request closed without merging**